feat(ruby): Add Profiling docs (#6650)
getsentry/sentry-docs: src/wizard/ruby/profiling-onboarding/ruby/2.configure-performance.md

---
name: Ruby
doc_link: https://discord.gg/zrMjKA4Vnz
support_level: beta
type: language
---

#### Configure Performance

> Sentry’s performance monitoring product has to be enabled in order for Profiling to work. To enable performance monitoring in the SDK:

Reviewer: ```suggestion
Sentry’s performance monitoring product must be enabled for Profiling to work. To enable performance monitoring in the SDK:
```
